pandas/tests/frame/test_apply.py

```diff
@@ -21,6 +22,38 @@
 from pandas.tests.frame.common import TestData
 
 
+def _get_cython_table_params(frame, func_names_and_expected):
+    """combine frame, functions from SelectionMixin._cython_table
+    keys and expected result.
+
+    Parameters
+    ----------
+    frame : DataFrame
+        A symmetrical DataFrame
+    func_names_and_expected : Sequence of two items
+        The first item is a name of a NDFrame method ('sum', 'prod') etc.
+        The second item is the expected return value
+
+    Returns
+    -------
+    results : list
+        List of three items (DataFrame, function, expected result)
+    """
+    table = pd.core.base.SelectionMixin._cython_table
+    if compat.PY36:
```

Comment:
can you make this first part a fixture in conftest

Reply:
then not sure you really need this function, can you not compute this directly give the name of the function and table?

Reply:
> can you make this first part a fixture in conftest

Not possible; this function is not a test function, so does not take fixtures

Reply:
> then not sure you really need this function, can you not compute this directly give the name of the function and table?

I'd want the the functions to take the lefthand side functions in ``_cython_table`` as input, so each one is tested individually. 

An alternative would be to pass the strings into the test functions (e.g. "sum"), and then get the relevant functions inside the test functions. That would mean that each test would run several subtests, which is a different kind of issue/problem.

So I think you have to choose between different kinds of ugliness here. I'd personally prefer my original solution. I'f someone has an idea for improvement, I'm wiling to give it another run, though

Reply:
this can easily BE a fixture, IOW you put this function in conftest.py and call it to create the params FOR a fixture.

Reply:
I've almost dried my brain looking into how to do this, I just can't see it... The function ``_get_cython_table_params`` can't take fixtures, as it's not a test function. Do you mean simply import it from conttest.py? (``from pandas.conftest import cython_table`` or similar) I assume that's not what you mean, tough.

Could you maybe spell it out how you see the implementation for me? I would be very grateful for that.

Reply:
I've added a fixture for cython table items. Can't use fixtures in ``_get_cython_table_params``, so do an import from conttest.py.

The build in CirceCi is a resourceerror, so unrelated to this PR.